diff --git a/OmniGibson/setup.py b/OmniGibson/setup.py
--- a/OmniGibson/setup.py
+++ b/OmniGibson/setup.py
@@ -73,7 +73,7 @@
         ],
         "primitives": [
             "ninja~=1.13.0",
-            "nvidia-curobo @ git+https://github.com/StanfordVL/curobo@d083916436f1f749cdd351bdf96c4c5df4e6d192",
+            "nvidia-curobo",
             "ompl @ https://storage.googleapis.com/gibson_scenes/ompl-1.6.0-cp310-cp310-manylinux_2_28_x86_64.whl",
         ],
         "eval": [
